ActivityMerger/commands.py

```diff
@@ -0,0 +1,30 @@
+# from datetime import datetime, timedelta
+import time
+import shutil
+import os
+import click
+
+from ActivityMerger import app
+
+
+@app.cli.command()
+@click.option("-a", "--all", "flush_all", is_flag=True)
+def flush_data(flush_all):
+    """
+    Delete folders that are older than 7 days
```

Comment:
that 7 may not be true if the config changes, should this comment mention the config var instead?

Reply:
Ah, good catch! Thanks.